packages/vm/lib/runBlock.ts

```diff
@@ -420,66 +378,17 @@ export async function rewardAccount(
 }
 
 /**
- * Generates the tx receipt and returns { txReceipt, encodedReceipt, receiptLog }
+ * Returns the encoded tx receipt.
  */
-export async function generateTxReceipt(
```

Comment:
This is also a breaking change. Maybe just deprecate it and leave it here?

Reply:
good idea, done in https://github.com/ethereumjs/ethereumjs-monorepo/pull/1185/commits/a6c622e486246765953425c06ef9a1b427bf536f